src/lib/data/audio.ts

```diff
@@ -383,9 +383,13 @@ export async function getAudioSourceInfo(item: {
     } else if (audioSource.type === 'assets') {
         audioPath = '/audio/' + audio.filename;
     } else if (audioSource.type === 'download') {
-        audioPath = audioSource.address + '/' + audio.filename;
+        let extraSlash = '/';
```

Comment:
Please make a more robust joinPaths function in stringUtils.ts?

  Look at:
https://stackoverflow.com/questions/29855098/is-there-a-built-in-javascript-function-similar-to-os-path-join